manager_rethink.go

```diff
@@ -70,6 +71,7 @@ type RethinkManager struct {
 	Policies map[string]Policy
 }
 
+// ColdStart loads all policies from rethinkdb without warming up the cache
```

Comment:
@arekkas I think what people do to get better performance is to warm up the cache by making a few queries to fill up the rethinkdb's cache. I thought it is called `ColdStart` here, because you don't warm up the rethinkdb cache first. Is this the case here?